config/rbac/role.yaml

```diff
@@ -219,6 +219,7 @@ rules:
   - patch
   - update
 - apiGroups:
+  - shipa.io
```

Comment:
with this ketch will be able to work with shipa's apps, not sure this is something we need